Library/Homebrew/style.rb

```diff
@@ -263,7 +263,10 @@ def shell_scripts
         HOMEBREW_REPOSITORY/"Dockerfile",
         *HOMEBREW_LIBRARY.glob("Homebrew/*.sh"),
         *HOMEBREW_LIBRARY.glob("Homebrew/shims/**/*").map(&:realpath).uniq
-                         .reject { |path| path.directory? || path.basename.to_s == "cc" },
+                         .reject do |path|
+                           path.directory? || path.basename.to_s == "cc" ||
+                             path == HOMEBREW_LIBRARY/"Homebrew/shims/ruby/bundle"
+                         end,
```

Comment:
```suggestion
                         .reject(&:directory?)
                         .reject do |path|
                           path.basename.to_s == "cc" || path == HOMEBREW_LIBRARY/"Homebrew/shims/ruby/bundle"
                         end,
```

and could consider splitting out all three. Also feel like it'd be nicer to move this out of the array.

Reply:
yeh, that'd work too